core/arch/arm/plat-hikey/conf.mk

```diff
@@ -23,5 +23,21 @@ endif
 CFG_NUM_THREADS ?= 8
 CFG_CRYPTO_WITH_CE ?= y
 CFG_WITH_STACK_CANARIES ?= y
+
 CFG_PL061 ?= y
 CFG_PL022 ?= y
+CFG_SPI ?= y
+
+ifeq ($(CFG_SPI_TEST),y)
+$(call force,CFG_SPI,y)
+endif
+
+ifeq ($(CFG_SPI),y)
+$(call force,CFG_PL061,y)
+$(call force,CFG_PL022,y)
+endif
+
+ifeq ($(CFG_PL061),y)
+arm64-platform-cflags		+= -DPLAT_PL061_MAX_GPIOS=160
```

Comment:
Oh, put it in `core-platform-cppflags` here. I misunderstood and thought it was in `arm.mk` or `core.mk`, hence the 'affect other platforms' question. Ok and thanks!